sherlock/notify.py

```diff
@@ -206,14 +219,15 @@ def update(self, result):
         
         # Output to the terminal is desired.
         if result.status == QueryStatus.CLAIMED:
+            self.countResults()
```

Comment:
for every social media in which the username exists, the function countResults() is called